docs/developers/serializers.rst

```diff
@@ -0,0 +1,187 @@
+.. _serializers:
+
+Serializers
+===========
+
+You can customize the serialization to and from Stores in Pootle.
+
+This allows you to make formatting or other changes before saving to disk, and
+conversely you can make changes to files before importing into Pootle.
+
+Serializers and deserializers are configured for a project. See the
+`django-admin`:"project_serializers" command for more information on configuring projects.
+
+To add your own serializers of deserializers you must add your own django application,
+and include it in your `INSTALLED_APPS` setting.
```

Comment:
this is code level configuration - are you arguing against us using a plugin system?